db/version_set.cc

```diff
@@ -4881,7 +4882,8 @@ Status VersionSet::ListColumnFamilies(std::vector<std::string>* column_families,
     if (!s.ok()) {
       return s;
   }
-  file_reader.reset(new SequentialFileReader(std::move(file), manifest_path));
+  file_reader.reset(new SequentialFileReader(std::move(file), manifest_path,
```

Comment:
What's the reason to pass `nullptr` here when creating the new `SequentialFileReader`?

Reply:
In some places we don't have access to ```IOTracer```. ```IOTracer``` object is created in ```DBImpl``` and then passed to different classes. ```VersionSet::ListColumnFamilies```  is static function so it doesn't have access to IOTracer. I will remove the nullptr here as  ```SequentialFileReader``` accepts default ```nullptr``` for IOTracer argument.

Reply:
Thanks for the explanation.